cli/lib/build.js

```diff
@@ -114,33 +112,34 @@ const checkModuleValid = ()=> {
   }
 }
 
-const buildAll =  async () => {
+const buildAll =  async (enableSourceMap) => {
   let start = 0;
   const len = moduleList.length;
+  const buildCmd = enableSourceMap ? 'build-width-sourceMap' : 'build';
 
   while (start < len) {
     const { moduleName, moduleDir } = moduleList[start];
     logInfo(`Building ${moduleName}`);
-    await spawnSync(npmCmd, ['run', 'build'], { env: process.env, cwd: moduleDir, stdio: 'inherit' });
+    await spawnSync(npmCmd, ['run', buildCmd], { env: process.env, cwd: moduleDir, stdio: 'inherit' });
```

Comment:
Pass `enableSourceMap` to the second argument as env variable to control build logic.So user don't have to add another 'build-width-sourceMap' script.